projects/go-lib/src/lib/components/go-table/go-table-column.component.ts

```diff
@@ -9,6 +9,7 @@ export class GoTableColumnComponent {
   @Input() field: string;
   @Input() title: string;
   @Input() width: number;
+  @Input() sortable?: boolean;
```

Comment:
Should we just default this to true and then we don't have to do the `undefined` check?